WORKSPACE

```diff
@@ -55,7 +55,7 @@ load("@rules_nodejs//nodejs:repositories.bzl", "nodejs_register_toolchains")
 # This applies to the resolved_toolchain
 nodejs_register_toolchains(
     name = "node16",
-    node_version = "16.5.0",
```

Comment:
Note that this change was required to for @angular/core - they require nodejs >=16.10.0